ios/yelpApp.xcodeproj/project.pbxproj

```diff
@@ -8,11 +8,13 @@
 
 /* Begin PBXBuildFile section */
 		00E356F31AD99517003FC87E /* yelpAppTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* yelpAppTests.m */; };
+		05B6A5D46C60487DB0C84412 /* Poppins-Medium.ttf in Resources */ = {isa = PBXBuildFile; fileRef = F9B0AF241B9646089F99EB44 /* Poppins-Medium.ttf */; };
```

Comment:
Check why this file keeps on updating

Reply:
As Discussed i will undo the changes of this file if it changes un intentionally before commiting and i will look for a solution for this.